sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/stringExpressions.scala

```diff
@@ -27,6 +29,8 @@ import org.apache.spark.sql.catalyst.util.ArrayData
 import org.apache.spark.sql.types._
 import org.apache.spark.unsafe.types.{ByteArray, UTF8String}
 
+import scala.util.control.NonFatal
```

Comment:
By the way, I'm wondering why this does not cause build errors. Usually, it's caught by build errors.

Anyway, could you run `./dev/scalastyle`?